### PR TITLE
[7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by JoyChukwukere**
html-forms
html-forms1

### CODE REVIEW

File: submissions/joyChukwukere/html-forms/html-forms.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                <option value="Damaged product">
+                <option value="Return generation">
+                <option value="Enquires">
+                <option value="Other issues">
+            </datalist> <br> <br>
+
+            <label for="message">Leave Your Message</label>  <br>
+            <textarea name="message" id="message" cols="50" rows="10"></textarea> <br>  <br>
+
+            <button onclick="alert('Thanks for contacting us!')">Submit</button>
```
Comment (adaugochi) [inline submissions/joyChukwukere/html-forms/html-forms.html:30]: Remove the js. This is html form task. No styling or JS should be added. follow instructions given in the task strictly

Comment (JoyChukwukere) [reply submissions/joyChukwukere/html-forms/html-forms.html:30]: Hi Adaa,
Thanks for the help the other day.
Please I do not understand the comment. I did only the things we were taught. By Js, do you mean <button onclick> part or the <text area>?

Comment (adaugochi) [reply submissions/joyChukwukere/html-forms/html-forms.html:30]: This is javascript `onclick="alert('Thanks for contacting us!')` . Remove it, it is not needed

Comment (JoyChukwukere) [reply submissions/joyChukwukere/html-forms/html-forms.html:30]: ok
Thanks

Comment (JoyChukwukere) [reply submissions/joyChukwukere/html-forms/html-forms.html]: I have removed it and i did add, commit and push. Please review and merge. Thanks!

**Pull request merged**